extras/rya.export/export.api/src/main/java/org/apache/rya/export/api/conf/MergeConfiguration.java

```diff
@@ -67,67 +61,35 @@
-            final String childHostname, final String childUsername, final String childPassword, final String childZookeepers,
-            final int childPort, final String childRyaInstanceName, final String childTablePrefix, final String childAuths, final String childInstanceType, final String childTomcatUrl, final DBType childDBType,
-            final MergePolicy mergePolicy, final boolean useNtpServer, final String ntpServerHost, final String toolStartTime) throws MergeConfigurationException {
+    protected MergeConfiguration(final Builder builder) throws MergeConfigurationException {
         try {
-            this.parentHostname = checkNotNull(parentHostname);
-            this.parentUsername = checkNotNull(parentUsername);
-            this.parentPassword = checkNotNull(parentPassword);
-            this.parentZookeepers = checkNotNull(parentZookeepers);
-            this.parentRyaInstanceName = checkNotNull(parentRyaInstanceName);
-            this.parentTablePrefix = checkNotNull(parentTablePrefix);
-            this.parentAuths = checkNotNull(parentAuths);
-            this.parentInstanceType = checkNotNull(parentInstanceType);
-            this.parentTomcatUrl = checkNotNull(parentTomcatUrl);
-            this.parentDBType = checkNotNull(parentDBType);
-            this.parentPort = checkNotNull(parentPort);
-            this.childHostname = checkNotNull(childHostname);
-            this.childUsername = checkNotNull(childUsername);
-            this.childPassword = checkNotNull(childPassword);
-            this.childZookeepers = checkNotNull(childZookeepers);
-            this.childRyaInstanceName = checkNotNull(childRyaInstanceName);
-            this.childTablePrefix = checkNotNull(childTablePrefix);
-            this.childAuths = checkNotNull(childAuths);
-            this.childInstanceType = checkNotNull(childInstanceType);
-            this.childTomcatUrl = checkNotNull(childTomcatUrl);
-            this.childDBType = checkNotNull(childDBType);
-            this.childPort = checkNotNull(childPort);
-            this.mergePolicy = checkNotNull(mergePolicy);
-            this.useNtpServer = checkNotNull(useNtpServer);
-            this.ntpServerHost = checkNotNull(ntpServerHost);
-            this.toolStartTime = checkNotNull(toolStartTime);
+            checkNotNull(builder);
+        } catch(final NullPointerException npe) {
+            throw new MergeConfigurationException("The configuration was passed a null builder.", npe);
+        }
+        try {
+            this.parentHostname = checkNotNull(builder.parentHostname);
+            this.parentUsername = checkNotNull(builder.parentUsername);
+            this.parentPassword = checkNotNull(builder.parentPassword);
+            this.parentRyaInstanceName = checkNotNull(builder.parentRyaInstanceName);
+            this.parentTablePrefix = checkNotNull(builder.parentTablePrefix);
+            this.parentTomcatUrl = checkNotNull(builder.parentTomcatUrl);
```

Comment:
It's needed for time synchronization.  When time sync'ing is used, an HTTP GET request is sent to a web server running on the remote machine (in this case Tomcat running on the parent or child machine) to determine the remote's local time.